### PR TITLE
React: Format USD fee in GasPrice

### PR DESCRIPTION
**Pull request by cehali**
# Summary
Add format currency for transaction fee in USD

Fixes: 0

## Checklist
- [x] Change is small and easy to review (please split big changes into multiple PRs)
- [x] Change is consistent with architecture
- [x] Change is consistent with test architecture
- [x] Change is consistent with naming conventions
- [x] New code is covered with tests
- [x] Tests related to old code are updated
- [x] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-react/src/core/utils/calculateTransactionFeeInUSD.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import {utils} from 'ethers';
+import {BigNumberish} from 'ethers/utils';
+import {safeMultiply} from '@universal-login/commons';
+import {formatCurrency} from './formatCurrency';
+
+export const calculateTransactionFeeInUSD = (usdAmount: BigNumberish, gasLimit: BigNumberish) =>
+  formatCurrency(safeMultiply(utils.parseEther(usdAmount.toString()), gasLimit));
```
Comment (rzadp) [inline universal-login-react/src/core/utils/calculateTransactionFeeInUSD.ts]: Would `calculateTransactionFeeInEUR` be different?
Consider rename to `calculateTransactionFee` ?

File: universal-login-react/src/core/utils/calculateTransactionFee.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import {utils} from 'ethers';
+import {BigNumberish} from 'ethers/utils';
+import {safeMultiply} from '@universal-login/commons';
+import {formatCurrency} from './formatCurrency';
+
+export const calculateTransactionFee = (usdAmount: BigNumberish, gasLimit: BigNumberish) =>
```
Comment (Marik-D) [inline universal-login-react/src/core/utils/calculateTransactionFee.ts]:
```suggestion
export const calculateTransactionFee = (usdGasPrice: BigNumberish, gasLimit: BigNumberish) =>
```

File: universal-login-react/src/core/utils/calculateTransactionFee.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  formatCurrency(safeMultiply(utils.parseEther(usdAmount.toString()), gasLimit));
```
Comment (Marik-D) [inline universal-login-react/src/core/utils/calculateTransactionFee.ts]:
```suggestion
  formatCurrency(safeMultiply(utils.parseEther(usdGasPrice.toString()), gasLimit));
```

File: universal-login-react/src/core/utils/calculateTransactionFee.ts (hunk 2 above)
Comment (rzadp) [reply universal-login-react/src/core/utils/calculateTransactionFee.ts]: Just `gasPrice`? It's not about USD

File: universal-login-react/src/core/utils/calculateTransactionFee.ts (hunk 3 above)
Comment (rzadp) [reply universal-login-react/src/core/utils/calculateTransactionFee.ts]: Just `gasPrice`? It's not about USD

**Pull request merged**